editor.planx.uk/src/pages/FlowEditor/components/Settings/tests/ServiceSettings/PublicLinks.test.tsx

```diff
@@ -121,7 +121,7 @@ describe("A team with a subdomain has an online, published service.", () => {
 
     user.click(copyButton);
 
-    expect(await screen.findByText("copied")).toBeVisible();
+    expect((await screen.findAllByText("copied"))[0]).toBeVisible();
```

Comment:
CI was throwing this error - 
<img width="1019" alt="image" src="https://github.com/user-attachments/assets/c21121d2-870f-44c2-b5de-5dbcf4f4320b">

This looks like it was catching the tooltip as well as the actual text.

<img width="201" alt="image" src="https://github.com/user-attachments/assets/4a2bfe60-a5c0-4f56-b2cb-42e57c5ba28a">